mmv1/third_party/terraform/resources/resource_bigquery_table.go

```diff
@@ -172,6 +173,23 @@ func bigQueryTableSchemaDiffSuppress(name, old, new string, _ *schema.ResourceDa
 	return eq
 }
 
+func bigQueryTableConnectionIdSuppress(name, old, new string, _ *schema.ResourceData) bool {
+	// API accepts connectionId in below two formats
+	// "{{project}}.{{location}}.{{connection_id}}" or
+	// "projects/{{project}}/locations/{{location}}/connections/{{connection_id}}".
+	// but always returns "{{project}}.{{location}}.{{connection_id}}"
+
+	if isEmptyValue(reflect.ValueOf(old)) || isEmptyValue(reflect.ValueOf(new)) {
```

Comment:
Do you know if you need this check? I didn't see many of our other DiffSuppress functions doing this, and from the logic you have below, it seems like this check might actually be redundant (ie. `matches` would be nil or the`==` would fail).

Reply:
I think you are right, but I'd like to exit here as it is clearer to read. It seems I saw several suppress functions that use this pattern.